build-support/bin/release.sh

```diff
@@ -7,20 +7,40 @@ set -e
 ROOT=$(cd $(dirname "${BASH_SOURCE[0]}") && cd "$(git rev-parse --show-toplevel)" && pwd)
 source ${ROOT}/build-support/common.sh
 
+# Note we parse some options here, but parse most at the bottom. This is due to execution order.
+# If the option must be used right away, we parse at the top of the script, whereas if it
+# depends on functions defined later in the script, we parse at the end.
+_OPTS="hdnftcloepqw3"
+
+while getopts "${_OPTS}"  opt; do
+  case ${opt} in
+    3) python_three="true" ;;
+    *) ;;  # skip over other args to be parsed later
+  esac
+done
+
+# Reset opt parsing's position to start
+OPTIND=0
+
 # Set the Python interpreter to be used for the virtualenv. Note we allow the user to
 # predefine this value so that they may point to a specific interpreter, e.g. 2.7.13 vs. 2.7.15.
-export PY="${PY:-python2.7}"
+default_interpreter="python2.7";
+if [[ "${python_three=false}" == "true" ]]; then
```

Comment:
I'm a bit confused by the mixing of `${python_three=false}` and `${python_three-false}`... is that intentional?

Reply:
Oof this was 100% a typo. Thanks for catching this! I also made the mistake in two other places. Here's to hoping CI still passes..